daux/docs/2000_Security/550_Integrating_OpenSSL.md

```diff
@@ -169,9 +169,12 @@ Here are the descriptions for the properties:
   * TLSv1.1
   * TLSv1.2
 
-All of the above algorithms support Java 6 and higher versions, except the TLSv1.2 supports Java 7 and higher versions. For 
-the `protocol` property, we recommend you to provide SSL or TLS with its version information, e.g., `TLSv1.2`. Note that if you
-configure only `SSL` or `TLS`, it will be converted to `SSLv3` and `TLSv1.2`.
+For the `protocol` property, we recommend you to provide SSL or TLS with its version information, e.g., `TLSv1.2`. Note that if you
+provide only `SSL` or `TLS` as a value for the `protocol` property, they will be converted to `SSLv3` and `TLSv1.2`, respectively.
+
+Except `TLSv1.2`, all of the listed algorithms above for the `protocol` property support Java 6 and higher versions. The protocol `TLSv1.2`, which is the default 
```

Comment:
This is not valid anymore AFAIK. The Oracle Java [cryptoroadmap](https://www.java.com/en/jre-jdk-cryptoroadmap.html) says the TLSv1.2 support was added to Java 6 in 2016 (6u115 b32).